protostar/cli/command.py

```diff
@@ -31,7 +31,7 @@ def directory(arg: str) -> Path:
         is_positional: bool = False
         is_required: bool = False
         is_array: bool = False
-        default: Optional[str] = None
+        default: Any = None
```

Comment:
Can we provide Union here instead of `Any`?

Reply:
The type of `default` depends on `Argument::type` and `Argument::is_array`. I'd rather mark it as `Any` to tell the developer that he can't rely on the type checker.